charts/argo-cd/values.yaml

```diff
@@ -999,10 +999,16 @@ server:
   # Find your domain with: kubectl describe --namespace=openshift-ingress-operator ingresscontroller/default | grep Domain:
   # If 'hostname' is an empty string "" OpenShift will create a hostname for you.
   route:
-    # -- Enable a OpenShift route for the Argo CD server
+    # -- Enable an OpenShift Route for the Argo CD server
     enabled: false
-    # -- Hostname of OpenShift route
+    # -- Openshift Route annotations
+    annotations: {}
+    # -- Hostname of OpenShift Route
     hostname: ""
+    # -- Termination type of Openshift Route
+    termination_type: Passthrough
```

Comment:
Are you sure that uppercase fits here? Before your change it was lower case and even in the OpenShift [here](https://docs.openshift.com/container-platform/3.11/architecture/networking/routes.html) docs it is lower case:
```yaml
apiVersion: v1
kind: Route
metadata:
  name: route-edge-secured 
spec:
  host: www.example.com
  to:
    kind: Service
    name: service-name 
  tls:
    termination: edge            
    key: |-                      
      -----BEGIN PRIVATE KEY-----
      [...]
      -----END PRIVATE KEY-----
    certificate: |-              
      -----BEGIN CERTIFICATE-----
      [...]
      -----END CERTIFICATE-----
    caCertificate: |-            
      -----BEGIN CERTIFICATE-----
      [...]
      -----END CERTIFICATE-----
---
apiVersion: v1
kind: Route
metadata:
  name: route-passthrough-secured 
spec:
  host: www.example.com
  to:
    kind: Service
    name: service-name 
  tls:
    termination: passthrough
---
apiVersion: v1
kind: Route
metadata:
  name: route-pt-secured 
spec:
  host: www.example.com
  to:
    kind: Service
    name: service-name 
  tls:
    termination: reencrypt        
    key: [as in edge termination]
    certificate: [as in edge termination]
    caCertificate: [as in edge termination]
    destinationCACertificate: |-  
      -----BEGIN CERTIFICATE-----
      [...]
      -----END CERTIFICATE-----
```